tests/foreman/ui/test_contentview.py

```diff
@@ -3675,36 +3640,32 @@ def test_positive_remove_cv_version_from_env_with_host_registered(self):
 
         @Steps:
 
-        1. Create a content view
-
+        1. Create a content view cv1
         2. Add a yum repo and a puppet module to the content view
-
         3. Publish the content view
-
         4. Promote the content view to multiple environment
            Library -> DEV -> QE
-
         5. Create an Activation key with the QE environment
-
         6. Register a content-host using the Activation key
-
-        7. Remove the content view version from QE environment
-
-        8. Verify if the affected Activation key and content-host are
-           functioning properly
+        7. Remove the content view cv1 version from QE environment.
+           The remove environment wizard should propose to replace the current
+           QE environment of cv1 by an other (as QE environment of cv1
+           is attached to a content-host),
+           choose DEV and content view cv1 as a replacement for Content-host
+           and for Activation key.
```

Comment:
As per discussion, need to add step to refresh the subscription from content-hosts..